tests/performance/config/perf-driver/measure/measure-fullDeploymentTime.yml

```diff
@@ -15,7 +15,23 @@ config:
     - name: fullDeploymentTime
       uuid: cfac77fceb244862aedd89066441c416
       desc: The time from the first HTTP request completion till the last deployment success
-      summarize: [mean_err]
+      summarize:
```

Comment:
What are percentiles for the `fullDeploymentTime` (which is 1 number)?

Reply:
Pointless 😂 
Let me remove them...

Reply:
Ah, by the way, since this is a re-usable fragment from all the tests, there are tests that _could_ have more than one `fullDeploymentTime` point (ex. when you repeat the tests). In that case I think it makes more sense?

Reply:
Hm. since we don't repeat tests right now, I'm in favour of removing it ;)

Reply:
We do (it's the `n-apps-1-instances` and `1-apps-n-instances` tests). But I can keep only the mean on them.